x-pack/plugins/observability/public/pages/alerts/containers/alerts_page/alerts_page.tsx

```diff
@@ -322,13 +330,18 @@ function AlertsPage() {
         </EuiFlexItem>
 
         <EuiFlexItem>
-          <AlertsTableTGrid
-            indexNames={indexNames}
-            rangeFrom={rangeFrom}
-            rangeTo={rangeTo}
-            kuery={kuery}
-            setRefetch={setRefetch}
-          />
+          <CasesContext
```

Comment:
This is the important change. To show the create case flyout, and eventually the case modal, the cases context needs initialization. Effectively this is the place where the flyout renders when invoked.